packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -73,9 +73,14 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
 
   onVrackServiceSelected(vrackService) {
     this.selectedSubnet = null;
-    this.subnets = angular.copy(vrackService.currentState.subnets);
+    this.subnets = angular
+      .copy(vrackService.currentState.subnets || [])
+      .map((subnet) => ({
+        ...subnet,
+        displayName: subnet.displayName ?? subnet.cidr,
+      }));
```

Comment:
**nitpick:** `Array.prototype.map` anyways returns a new array with new objects. So `angular.copy` is obsolete.